ts/nni_manager/training_service/remote_machine/extends/linuxCommands.ts

```diff
@@ -124,6 +124,12 @@ class LinuxCommands extends OsCommands {
             command = `bash '${script}'`;
         } else {
             script = script.replace(/"/g, '\\"');
```

Comment:
Looks like resolving escape characters but can't understand what exactly it is doing.
Please describe what kind of script are you trying to escape or unescape. And if it is possible, we should avoid handling escape manually.

Reply:
This wants to solve the situation that `bash -c` nested `echo "some command \\"something\\""`. This happens in sharedstorage mount command.